test/test.tf

```diff
@@ -63,7 +63,7 @@ module "test_nodes_sydney_channels" {
 }
 
 module "test_lb_sydney" {
-  source                    = "github.com/aeternity/terraform-aws-api-loadbalancer?ref=master"
+  source                    = "github.com/aeternity/terraform-aws-api-loadbalancer?ref=test-on-multiple-terraform-versions"
```

Comment:
will be fixed after https://github.com/aeternity/terraform-aws-api-loadbalancer/pull/12/ merge as master have issues